network/endpoint.go

```diff
@@ -290,7 +290,7 @@ func (ep *endpoint) attach(sandboxKey string) error {
 
 	ep.SandboxKey = sandboxKey
 
-	logger.Info("Attached endpoint to sandbox", zap.String("id", ep.Id), zap.String("sandboxKey", sandboxKey))
+	logger.Info("Attached endpoint to sandbox", zap.String("id", ep.Id))
```

Comment:
looks like zap.string("id", xxx) is reused in lots of places. maybe worth to add a common func for zap field for id going forward.  just an idea. 

func ID(id string) zap.Field {
	return zap.String("id", id)
}